ArduPlane/mode_autoland.cpp

```diff
@@ -152,14 +164,39 @@ bool ModeAutoLand::_enter()
 void ModeAutoLand::update()
 {
     plane.calc_nav_roll();
+
+    // Constrain the roll limit if climb before turn options are set when below that altitude,
+    //that way if something goes wrong the plane will
+    // eventually turn back and go to landing waypoint instead of going perfectly straight. This also leaves
+    // some leeway for fighting wind.
+    check_altitude();
```

Comment:
your only using this function in one spot, might as a well inline it.

Reply:
cant use RTL_CLIMB_MIN.....its a delta...want an absolute like RTL_ALTITUDE + CLIMB before turn option....in case AUTOLAND_WP_ALT (a final approach alt) wont cut it because of obstacles....will inline it and use flaot